.travis.yml

```diff
@@ -1,21 +1,24 @@
-sudo: false
-language: java
-jdk:
-  - openjdk6
-  - openjdk7
-  - oraclejdk7
+sudo: required
```

Comment:
I tried the test with the old `sudo: false` first and got the standard "you need to be root" message out of `apt-get`, so it seems it is required. Additionally I think we need to be explicit: my read of the docs [here](http://docs.travis-ci.com/user/build-configuration/#Installing-Packages-Using-apt) is that Travis will automatically infer our need for `sudo` *if* we use `apt-get` in a `.travis.yml` section, but our `apt-get` is hidden in a shell script.

I couldn't find any note of a downside in the docs -- at least, they make it sound like this is something that can be turned on whenever we need it, and it's totally fine because everything is in containers.